app/assets/stylesheets/components/editorToolbar.scss

```diff
@@ -0,0 +1,45 @@
+.editor-toolbar {
+  display: flex;
+  background: var(--base-0);
+  padding: var(--su-2) var(--content-padding-x);
+
+  &#overflow-menu {
+    position: asbolute;
+    background: revert;
```

Comment:
Oh I did not know that `revert` value! Always thought about `unset`. Do you know the difference?

Also out of curiosity, why is it needed here?

Reply:
[This probably does a better job of explaining it](https://developer.mozilla.org/en-US/docs/Web/CSS/revert), but it essentially just pops the property back to the browser default / if a user has a custom stylesheet. In this case `unset` would be fine, I just default to `revert` a lot 🤷‍♀️ 

It's not actually the right thing for here, so it'll be changing, but originally when I was messing about I used it to remove the gray background the overflow inherited from the `editor-toolbar` class